.github/workflows/ci.yml

```diff
@@ -22,3 +27,30 @@ jobs:
 
       - name: Build and test frontend
         run: yarn build
+
+      - name: Upload package for e2e tests
+        uses: actions/upload-artifact@v3
+        with:
+          name: dist
```

Comment:
We could use a nice name here, like compiled-plugin. That would look better if you download it from the actions page.

Reply:
renamed it to `plugin-build-result`